runtime/shared_common/ScopeManagerImpl.cpp

```diff
@@ -96,10 +96,6 @@ SH_ScopeManagerImpl::scHashEqualFn(void* left, void* right, void *userData)
 		Trc_SHR_SMI_scHashEqualFn_Exit2();
 		return 0;
 	}
-	if (J9UTF8_DATA(utf8left)==NULL || J9UTF8_DATA(utf8right)==NULL) {
-		Trc_SHR_SMI_scHashEqualFn_Exit1();
```

Comment:
The tracepoint `Trc_SHR_SMI_scHashEqualFn_Exit1` should be marked `Obsolete` now that it is no longer used.

Reply:
What does it mean to mark a tracepoint as `Obsolete`?

Reply:
In `runtime/shared_common/j9shr.tdf` add the word `Obsolete` after `Trc_SHR_SMI_scHashEqualFn_Exit1`.
See also https://github.com/eclipse-openj9/openj9/blob/master/doc/diagnostics/AddingTracepoints.md.

Reply:
Will do, thanks for the tip!